keyboards/handwired/oem_ansi_fullsize/info.json

```diff
@@ -0,0 +1,119 @@
+{
+  "keyboard_name": "oem_ansi_fullsize",
+  "url": "https://github.com/johnstcn/qmk",
+  "maintainer": "johnstcn",
+  "width": 22,
+  "height": 6,
+  "layouts": {
+    "LAYOUT": {
+      "layout": [
+        [
+          "KC_ESC",
```

Comment:
This file is incorrect. you need key positions, not keycodes, here.

Reply:
Done